src/reference/modules/usage/pages/variables.adoc

```diff
@@ -783,6 +783,11 @@ Conditions can be combined using boolean operators:
 - `.` for and
 - `(` and `)` for grouping
 
+This is an example about how to define two generic methods, one for installing `python3-jinja2` on `CentOS 8` (with the OS system variable, see below), and one for installing `python-jinja2` on `CentOS 6` and `7`:
```

Comment:
The prefered way for OS condition is to use the form

Reply:
That allows chosing OS name and version

Reply:
but you can't say "not that" with it ?

Reply:
I will demo both

Reply:
Indeed!